src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-index {
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  padding: 0;
+  margin-right: -20px;
```

Comment:
The `margin-right: -20px;` might cause layout issues by pulling the last item in the `.nav-index` list too far to the left. Consider adjusting this value to ensure proper alignment.